packages/cloud-cognitive/src/components/AboutModal/_about-modal.scss

```diff
@@ -73,9 +65,11 @@ $block-class: #{c4p-settings.$pkg-prefix}--about-modal;
   margin-left: $spacing-03;
```

Comment:
I think this border-left should update the token to `$border-strong` or `$border-strong-01`.